### PR TITLE
dune fmt: use ocamlformat.0.11.0

### PR DESCRIPTION
**Pull request by craigfe**
I suggest use of OCamlformat :slightly_smiling_face:

Let me know if there are any particularly-objectionable diffs.

### CODE REVIEW

File: lib/tests/misc.ml [hunk 1]
```diff
@@ -88,41 +92,46 @@ let strict_table () =
...
+               ]);
+         main_menu_item "View" (fun _ ->
+             bot $= Lwd.return (string "<View>");
+             Lwd.return Ui.empty);
+         main_menu_item "Edit" (fun _ ->
+             bot $= Lwd.return (string "<Edit>");
+             Lwd.return Ui.empty);
+       ];
+  bot
+  $= Lwd_utils.pack Ui.pack_y
```
Comment (let-def) [inline lib/tests/misc.ml:131]: It is funny to break the line around the (small) lhs of a binary operator. But that's fine for me.

**Comment by let-def** [comment]
I am not used to ocamlformat, but that's unquestionably a good practice for team work. Thanks for the patch.

File: lib/tests/misc.ml (hunk 1 above)
Comment (craigfe) [reply lib/tests/misc.ml:131]: Indeed, @Julow spotted this before I did and we're all agreed that this is sub-optimal. OCamlformat has an ongoing battle with infix operators of various types. I have a few half-baked proposals for how to fix it, but if you have any thoughts yourself I'm sure the team would love to hear them.

File: src/main.ml [hunk 2]
```diff
@@ -88,236 +96,232 @@ let reduce_string_table table width =
...
+             ];
+           Lwd_utils.pack Ui.pack_x
+             [
+               Lwd_utils.pack Ui.pack_y
+                 [ Lwd.bind (Lwd.get width) (reduce_string_table log_lines) ]
+               |> W.vscroll_area ~state:(Lwd.get scroll_state)
+                    ~change:set_scroll
+               |> Lwd.map (fun ui ->
+                      ui |> Ui.resize ~w:0 ~sw:1
+                      |> Ui.size_sensor (fun w _ -> update_if_changed width w));
```
Comment (let-def) [inline src/main.ml:285]: Another minor remark: `|>` are often put on different lines (to visually reflect the pipeline).
This seems to be the case elsewhere, but here only the second one is put on its own line (?!).

File: lib/tests/misc.ml (hunk 1 above)
Comment (let-def) [reply lib/tests/misc.ml:131]: That's already good enough for me :D. I have no idea how to fix that, my doctor told me to stay away from `Format` ¯\_(ツ)_/¯.

File: src/main.ml (hunk 2 above)
Comment (craigfe) [reply src/main.ml:285]: Indeed, I think OCamlformat will essentially word-wrap them to 80 characters. I suspect the desired behaviour is to split them onto separate lines as soon as they spill over from a single line, but then there is the question of being consistent with user-defined sequencing like `>>=` and `>|=`.

Comment (Julow) [reply src/main.ml:285]: That's not great indeed.

Comment (Julow) [reply src/main.ml:285]: Wait ! There is an option for that. You could add `break-infix = fit-or-vertical` in `.ocamlformat`.

**Pull request merged**